server/views/pages/confirm-instructions.njk

```diff
@@ -44,23 +56,22 @@
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-one-half">
-      <div class="govuk-form-group" data-module="moj-add-another">
```

Comment:
this change goes against the example https://design-patterns.service.justice.gov.uk/components/add-another/, is the example bad or does this mess with the formatting of the page?

Reply:
Must admit we didn't look at the design patterns. We went off the prototype:
[prototype](https://manage-a-workforce-main.apps.live.cloud-platform.service.justice.gov.uk/allocation-notes)
which does it more or less like this i.e. the moj-add-another is wrapped around the section of the form where you add another, rather than the whole form.

We did make this change to try to fix the page layout. We wanted to run it past Leigh before going live with it, as the spacing does not match the prototype in every regard.